### ISSUE TITLE
Chromatic calculator paste doesn't work

### ISSUE THREAD

**Issue by mcmanustfj** [Lailloken/Lailloken-UI#208]
When pressing ctrl+V in the 'str' field in the chromatic calculator website after opening it through the UI, it tabs through all the stats but doesn't actually paste anything to it

**Comment by Lailloken** [comment]
Yeah, somebody told me about this as I was going to bed. It's minor and there's a release scheduled for later today, so I didn't want to push a late hofix release.

**Comment by Lailloken** [comment]
fixed in v1.30.3
